CONTRIBUTING.md

```diff
@@ -48,12 +48,15 @@ request and branch commits.
 
 This is a rough outline of what a contributor's workflow looks like:
 
-- Create a branch from where you want to base your work (usually master).
-- Make your changes and arrange them in readable commits.
-- Make sure your commit messages are in the proper format (see below).
-- Push your changes to the branch in your fork of the repository.
-- Make sure all linters and tests pass, and add any new tests as appropriate.
-- Submit a pull request to the original repository.
+1. Open an issue against this repository to propose your change.
+1. Consider writing a [one-pager or design doc](design/) if your proposal seems
+   complex or controversial.
+1. Create a branch from where you want to base your work (usually master).
```

Comment:
This is already merged, so I'll just leave this comment without need for follow-up.

I think "create a branch" and "push your changes to the branch" are not necessary distinctions for users contributing to this project.  That is naturally the flow through which Github would take a user.  This is purely exaggeration, but a user coming across these instructions outside of Github may need `git` instructions as well.

Would "1. Arrange your changes in readable commits on a branch in your GitHub fork" have sufficed?

We could link to branching and forking docs if the original bullets still seem necessary.

I agree with [your stance on approachability](https://github.com/crossplaneio/crossplane/pull/535#issuecomment-501458787), so I find less bullets are better.

Reply:
I agree - I've raised https://github.com/crossplaneio/crossplane/issues/536 to track this. Everything but the first two items in this list already existed - I just changed them from an unnumbered to a numbered list. I was tempted to fiddle more with this document but decided to avoid too much driveby refactoring.